src/front/spv/function.rs

```diff
@@ -341,14 +341,10 @@ impl<I: Iterator<Item = u32>> super::Parser<I> {
 
                     let mut arg = arg.clone();
                     if ep.stage == crate::ShaderStage::Fragment {
-                        if let Some(crate::Binding::Location {
-                            interpolation: ref mut interpolation @ None,
-                            ..
-                        }) = arg.binding
-                        {
-                            *interpolation = Some(crate::Interpolation::Perspective);
-                            // default
-                        }
+                        crate::proc::apply_default_interpolation(
+                            &mut arg.binding,
```

Comment:
can this look like:
```rust
arg.binding.apply_default_interpolation(&module.types[arg.ty].inner);
```
?

Reply:
It can, but because we can't define methods on `Option<Binding>`, the code looks like this:

```
if let Some(ref mut binding) = arg.binding {
    binding.apply_default_interpolation(&module.types[arg.ty].inner);
}
```

I can make that change.